tmitchell/exercise.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/python3
+"""Run an interactive shell, users can view and edit a phone book and then save it to file."""
+class Person:
+    """Entry in phonebook able tyo store and display information"""
+    def __init__(self, name, hnumber, mnumber):
+        self.name = name
+        self.hnumber = hnumber
```

Comment:
I would use home_number as opposed to hnumber. Verbosity isn't necessarily bad within reason.